src/ExpressiveInstaller/Resources/config/container-auryn.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+use Auryn\Injector;
+use Northwoods\Container\InjectorContainer;
+use Psr\Container\ContainerInterface;
+
+// Load configuration
+$config = require __DIR__ . '/config.php';
+
+// Create injector
+$injector = new Injector();
+
+// Configuration will be used globally
+$injector->share('config')->delegate('config', function () use ($config) {
+    // Auryn requires that all injections resolve to an object.
+    // Wrap the config array with an ArrayObject to appease the gods.
```

Comment:
[aura.da still does it](https://github.com/zendframework/zend-expressive-skeleton/blob/d6d44f34dfc9a1eef1cecc8c99538ff2df982662/src/ExpressiveInstaller/Resources/src/ExpressiveAuraConfig.php#L54)